openfasoc/generators/ldo-gen/flow/scripts/openfasoc/pre_global_route.tcl

```diff
@@ -3,7 +3,7 @@ source $::env(SCRIPTS_DIR)/openfasoc/create_rVREG.tcl
 
 # NDR rules
 source $::env(SCRIPTS_DIR)/openfasoc/add_ndr_rules.tcl
-
+global_connect
```

Comment:
can you add a comment for this global connect

Reply:
Yes. Will add comment for the global connect statement